src/_guides/openapi/specification/v3.1/understanding-structure/basic-structure.md

```diff
@@ -0,0 +1,317 @@
+
+  requestBodies:
+    TripRequest:
+      description: A request body for creating a new trip.
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Trip'
+
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: X-API-KEY
+
+  parameters:
+    pageParam:
+      in: query
+      name: page
+      required: false
+      schema:
+        type: integer
+        default: 1
+        description: The page number for pagination.
+```
+
+Understanding `$ref` is a large topic, and we'll get into it later, but just understand for now that this is where reusable components of all sorts are stored.
+
+<!-- TODO Link advanced / splitting documents into multiple parts -->
+
+Another important type of component is the `securitySchemes` section. OpenAPI supports several authentication types, but here are a few examples:
+
+```yaml
+components:
+  securitySchemes:
+    ApiKeyHeader:
+      type: apiKey
+      in: header
+      name: X-API-Key
+    
+    JWT:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+   
+    OAuth2ReadWrite:
+      type: oauth2
+      flows:
+        authorizationCode:
+          scopes:
+            read: Grants read access
+            write: Grants write access
+          authorizationUrl: https://example.com/oauth/authorize
+          tokenUrl: https://example.com/oauth/token
+          refreshUrl: https://example.com/oauth/refresh
+```
+
+This is just a few examples of various types of security schemes that can be defined, but defining them alone doesn't do anything. They need to be referenced by the `security` object.
+
+### 6. Security Object
+
+The top-level `security` object specifies the security schemes that apply globally to the API, so if an entire API uses an API key or OAuth2 you might have:
+
+```yaml
+security:
+  - apiKey: []
+  - oauth2:
+    - read
+    - write
+```
+
+You can get into path specific overrides and various complex "and" situations with more [advanced security functionality](../advanced/security.md).
+
+### 7. Webhooks Object
+
+```yaml
+webhooks:
+  newBooking:
+    post:
+      operationId: new-booking
+      summary: New Booking
+      description: |
+        Subscribe to new bookings being created, to update integrations for your users.  Related data is available via the links provided in the request.
+      tags:
+        - Bookings
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Booking'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully.
+```
+
+### 8. Tags Object
+
+You may have spotted the `tags` keyword in the paths and webhooks, and those are referencing tags defined in the top-level `tags` object. The tag name is used to group related operations together. Each tag has a name and an optional description.
```

Comment:
:tipping_hand_man: The `paths` objects in this guide doesn't have a `tags` keyword might be good to add it to match the introduction sentence of this paragraph.